### DISCUSSION TITLE
Domain Rotation Calculation

### DISCUSSION THREAD

**Discussion by fxm200013** [Colvars/colvars#696] [General]
Hello, I need help. I have been using colvars for a while to measure the rotation of a domain in a protein but my results are not being consistent. I have use for the most part the rotationangle and the spinangle function but I don't seem to correlate what I see visually in vmd with the plots I obtained in colvar. Do you have any tips on how to approach this problem? Or where should I start to trouble shoot? Any help is more than welcome!

![Colvars-dashboard](https://github.com/Colvars/colvars/assets/112004747/c41eac1f-eb3b-450e-9bfb-6b856db69896)

![Rotation_github](https://github.com/Colvars/colvars/assets/112004747/4ee0614b-2255-462c-9b8b-c5709cec7f35)
I align my trajectory to my reference crystal structure to be able to measure the rotation

**Comment by jhenin** [comment]
Hi! A few tips to troubleshoot this using the Dashboard:
- just in case, double-check the axis you are using in the tilt and spinAngle functions. The default is z.
- you can visualize the overall rotation operator (your colvar named Vector) by selecting it and clicking "Show Rotation", which will show you the actual optimal rotation axis between the current frame and the reference coordinates.
- you can rotate the protein manually in VMD (Mouse->move->Molecule then shift-click), and press F5 to refresh the computed value of all the colvars. This way, you can test hypotheses about what orientation corresponds to what value.
